conf/logs.json

```diff
@@ -1811,6 +1811,20 @@
       ]
     }
   },
+  "fleet:results":{
```

Comment:
Do you open to rename the schema to `fleet:audit` to be more descriptive? The new feature in fleet 2.1.0 is to end audit logs.

Reply:
True however this is specifically looking at the output of `filesystem_result_log_file` which is the result of the query. 

There is a secondary kinesis output of `filesystem_status_log_file` which is not used in this example as it contains basic processing info and nothing to alert on. 

I have no strong preference on this however I felt `results` in this case matched closer to the Kolide output name.